tasks/src/translation/data.ts

```diff
@@ -52,7 +52,7 @@ const taskData: TaskData = {
 			id:          "t5-base",
 		},
 	],
-	summary:      "Machine translation is the task of translating text from one language to another.",
+	summary:      "Translation is the task of transcribing text from one language to another.",
```

Comment:
transcribe means to convert speech into text to I'd rather keep the old one

Reply:
Great! Thanks for this! I was going to be making this mistake very frequently haha